packages/web/src/scss/components/Accordion/index.html

```diff
@@ -0,0 +1,197 @@
+{{#> layout/plain }}
+
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Default accordion</h2>
+
+  <div class="mb-400">
+
+    <section
+      id="accordionExample"
+      class="Accordion"
+      data-toggle="accordion"
+    >
+
+      <!-- Accordion Item -->
+      <article
+        id="accordionExample_article_0"
+        class="Accordion__item"
+      >
+        <h3
+          id="accordionExample_article_0_header"
+          class="Accordion__itemHeader"
+        >
+          <button
+            type="button"
+            class="Accordion__itemToggle"
+            data-toggle="collapse"
+            data-target="accordionExample_article_0_collapse"
+            aria-controls="accordionExample_article_0_collapse"
+          >
+            <!-- toggle -->
+            Accordion header #0
+            <!-- // toggle -->
+          </button>
+          <span class="Accordion__itemSlot">
+            <!-- slot -->
+            <a href="#">Link</a>
+            <span class="Pill Pill--selected">3</span>
+            <svg class="Icon Accordion__itemIcon" width="24" height="24">
+              <use xlink:href="/icons/svg/sprite.svg#chevron-down" />
+            </svg>
+            <!-- // slot -->
+          </span>
+        </h3>
+        <div
+          id="accordionExample_article_0_collapse"
+          class="Accordion__itemCollapse"
+          data-parent="accordionExample"
+          aria-labelledby="accordionExample_article_0_header"
+        >
+          <div
+            class="Accordion__itemBody Collapse__content"
```

Comment:
There is a space of 32px around the content in the design. I am not sure if it is part of an example or design.

Reply:
<img width="340" alt="Screenshot 2022-11-16 at 20 12 49" src="https://user-images.githubusercontent.com/2481010/202271873-5e0dfb1d-6aaf-49a5-9def-1e53e633828b.png">

Reply:
The specification is that the content should not have internal padding. So i keep that without any offset

Reply:
Ok, you are right. I've rechecked it with the ticket.